Core/src/org/sleuthkit/autopsy/datasourceprocessors/VolatilityProcessor.java

```diff
@@ -229,8 +229,8 @@ private void lookupFiles(Set<String> fileSet, String pluginName) {
             if (fileName.length() < 1) {
                 continue;
             }
-           // if there is no extension, add a wildcard to the end
-            if (fileName.contains(".") == false) {
+           // if there is no extension, add a wildcard to the end except for handles as it leads to duplicates and missed files
+            if (fileName.contains(".") == false && pluginName != "handles") {
```

Comment:
Do you have examples of what was causing problems in handles? Should they be filtered out?

Reply:
Users\Public\Desktop pulls back Desktop.ini
System Volume Information\{e15d4b08-1598-11e8-93e6-080027b5e733}{3808876b-c176-4e48-b7ae-04046e6cc752} is not found 

Several others that do not have file extensions were not found as well.

Reply:
Ah.  Maybe this logic should then first try to lookup the passed in name w/out the wildcard and then we only add it if it failed to find anything.